recipes/pydoe/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "0.3.8" %}
+
+package:
+  name: pydoe
+  version: {{ version }}
+
+source:
+    fn: pyDOE-afc5949f01c010a26764adba9624871d51797270.tar.gz
+    url: https://github.com/tisimst/pyDOE/archive/afc5949f01c010a26764adba9624871d51797270.tar.gz
+    sha1: 54f87d106c3f49e1b4cff17a31ef8a9e0142ec5b
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+  number: 0
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - scipy >=0.13
+    - numpy >=1.8
+
+test:
+  imports:
+    - pyDOE
+
+about:
+  home: https://github.com/tisimst/pyDOE
+  license: BSD 3-Clause
+  summary: Design of experiments for Python
+  description: The pyDOE package is designed to help the scientist, engineer, statistician, etc., to construct appropriate experimental designs.
```

Comment:
This is what I did. The longer version is moved to the description section. It is OK for the description to be more verbose. This the field that GUI tools use to provide the user with a more detailed description of what the package does.

 The actual sentences are those of the author from the docs.